src/config/NEUTRONRCF745/config.h

```diff
@@ -0,0 +1,152 @@
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+
+#define USE_BARO
+#define USE_BARO_DPS310
+
+#define USE_MAG
+#define USE_MAG_QMC5883
+
+#define BEEPER_PIN           PD2
+
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PB4
+#define MOTOR4_PIN           PB5
+
+#define MOTOR5_PIN           PD12
+#define MOTOR6_PIN           PD13
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
+
+#define LED_STRIP_PIN        PA8
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART4_TX_PIN         PA0
+#define UART6_TX_PIN         PC6
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PA1
+#define UART6_RX_PIN         PC7
+#define UART7_RX_PIN         PE7
+#define UART8_RX_PIN         PE0
+#define I2C1_SCL_PIN         PB8
+#define I2C2_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB9
+#define I2C2_SDA_PIN         PB11
+#define LED0_PIN             PC13
+
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PC10
+#define SPI4_SCK_PIN         PE2
+
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PC11
+#define SPI4_SDI_PIN         PE5
+
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PC12
+#define SPI4_SDO_PIN         PE6
+
+#define CAMERA_CONTROL_PIN   PB3
+#define ADC_VBAT_PIN         PC3
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC2
+#define ADC_EXTERNAL1_PIN    PC1
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PE4
+#define GYRO_1_EXTI_PIN      PD0
+#define GYRO_2_EXTI_PIN      PD8
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PB12
+
+// TIM1, CH1, PA8, 1
+// TIM2, CH2, PB3, 2
+// TIM8, CH2N, PB0, 1, 3, 8
+// TIM8, CH3N, PB1, 1, 3, 8
+// TIM3, CH1, PB4, 3
+// TIM3, CH2, PB5, 3
+// TIM4, CH1, PD12, 4
+// TIM4, CH2, PD13, 4
+// TIM3, CH3, PC8, 3, 8
+// TIM3, CH4, PC9, 3, 8
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 1, PB3 , 1,  0) \
+    TIMER_PIN_MAP( 2, PB0 , 3,  0) \
+    TIMER_PIN_MAP( 3, PB1 , 3,  0) \
+    TIMER_PIN_MAP( 4, PB4 , 1,  0) \
+    TIMER_PIN_MAP( 5, PB5 , 1,  0) \
+    TIMER_PIN_MAP( 6, PD12, 1,  0) \
+    TIMER_PIN_MAP( 7, PD13, 1,  0) \
+    TIMER_PIN_MAP( 8, PC8 , 1,  1) \
+    TIMER_PIN_MAP( 9, PC9 , 1,  2)
+
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_ALIGN CW180_DEG
+#define MAG_ALIGN_YAW 1800
+//TODO #define MAG_BUSTYPE I2C
+#define MAG_I2C_INSTANCE (I2CDEV_2)
+
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+//TODO #define DSHOT_BIDIR ON
+//TODO #define MOTOR_PWM_PROTOCOL DSHOT600
```

Comment:
Please remove

Reply:
DSHOT300 or DSHOT600 default is set depending on MCU upon having USE_DSHOT enabled